.github/workflows/local_unit_test.yml

```diff
@@ -31,11 +31,24 @@ jobs:
 
       - name: Calculate coverage 
         run: make lcov | tee lcov_out.txt
-
+ 
       - name: Confirm 100% line coverage
         run: |
           if [[ `grep -A 3 "Overall coverage rate" lcov_out.txt | grep lines` != *"100.0%"* ]]; then
             grep -A 3 "Overall coverage rate" lcov_out.txt
             echo "Lacks 100.0% line unit test coverage"
             exit -1
           fi
+
+      - name: Confirm absolute line coverage
+        run: |
+          missed_branches=4
```

Comment:
why does this start at 4? Can we add a comment explaining that?

Reply:
Added a comment 👍